### DISCUSSION TITLE
How do I xdebug when running Pest tests in PHP Storm

### DISCUSSION THREAD

**Discussion by cheesegrits** [beyondcode/herd-community#630] [General]
I'm using the automatic xdebug detection, with PHP Storm.

How do I get xdebug to work when running Pest tests?

**Comment by mpociot** [comment]
You can run `herd debug ./vendor/bin/pest`

**Reply by dcaswel** [reply]
@mpociot I was having an issue with this as well. It would be really nice if we could just use the run/debug tools from within PhpStorm. Do you know of a way to do that or is anyone working on making that possible?
